collection.go

```diff
@@ -1061,9 +1088,18 @@ func (c *Collection) UpdateMany(ctx context.Context, filter, update interface{},
 		upsert = *opt.Upsert
 	}
 
+	// get arrayFilters
```

Comment:
Sorry very pedantic, but "get array filters" sounds better to me. Same should be used above and below.

Reply:
Done.